### DISCUSSION TITLE
Renault Kadjar 2016 tried to update my software to v9 and now i have some problems. I managed to downgrade to v2 but i had some errors and now i have some issues. Can t pass this screen now. Sound is ok parking sensors are working.

### DISCUSSION THREAD

**Discussion by Gorbanezu** [cedricp/ddt4all#913] [Q&A]

**Comment by Gorbanezu** [comment]
![WhatsApp Image 2023-09-24 at 13 12 43 (3)](https://github.com/cedricp/ddt4all/assets/145204716/82fa6715-3f22-495a-9f30-f896a1cde7bf)
![WhatsApp Image 2023-09-24 at 13 12 43 (2)](https://github.com/cedricp/ddt4all/assets/145204716/b452110b-6963-45ae-976e-f0f0fabd28c0)
![WhatsApp Image 2023-09-24 at 13 12 43 (1)](https://github.com/cedricp/ddt4all/assets/145204716/4f655ea5-80fe-485b-889f-ab6d618a45e9)
![WhatsApp Image 2023-09-24 at 13 12 43](https://github.com/cedricp/ddt4all/assets/145204716/403bf4d3-b10e-4204-94e0-6892459f8a3e)
![WhatsApp Image 2023-09-24 at 13 12 42 (1)](https://github.com/cedricp/ddt4all/assets/145204716/e5577797-7dd7-44b8-ad2d-4d3eb8642b92)
![WhatsApp Image 2023-09-24 at 13 12 42](https://github.com/cedricp/ddt4all/assets/145204716/6a058a46-df62-424b-91c7-d19e4a84b4ac)

**Comment by matzel82** [comment]
@all I work at a Renault garage. Not every hardware is compatible to every software..

**Comment by Gorbanezu** [comment]
> @ALL I work at a Renault garage. Not every hardware is compatible to every software..

So what can i do to repair my mistake?

**Comment by Gorbanezu** [comment]
can i do something with virgin mode?

**Comment by shrlnm** [comment]
@Gorbanezu How did you revert v2 from the v9? 
You are luky man!)
Try to use this instruction for recovering the configuration. https://www.drive2.ru/b/643019689764615344/

**Comment by Gorbanezu** [comment]
> @Gorbanezu How did you revert v2 from the v9? You are luky man!) Try to use this instruction for recovering the configuration. https://www.drive2.ru/b/643019689764615344/

I downgraded to v7 and after that a modified version of 3.3 and after some other modified versions i managed to downgrade to 2.2 but now this problem

**Comment by AndyIreland** [comment]
> ![WhatsApp Image 2023-09-24 at 13 12 43 (3)](https://user-images.githubusercontent.com/145204716/270158008-82fa6715-3f22-495a-9f30-f896a1cde7bf.jpeg) ![WhatsApp Image 2023-09-24 at 13 12 43 (2)](https://user-images.githubusercontent.com/145204716/270158011-b452110b-6963-45ae-976e-f0f0fabd28c0.jpeg) ![WhatsApp Image 2023-09-24 at 13 12 43 (1)](https://user-images.githubusercontent.com/145204716/270158014-4f655ea5-80fe-485b-889f-ab6d618a45e9.jpeg) ![WhatsApp Image 2023-09-24 at 13 12 43](https://user-images.githubusercontent.com/145204716/270158016-403bf4d3-b10e-4204-94e0-6892459f8a3e.jpeg) ![WhatsApp Image 2023-09-24 at 13 12 42 (1)](https://user-images.githubusercontent.com/145204716/270158017-e5577797-7dd7-44b8-ad2d-4d3eb8642b92.jpeg) ![WhatsApp Image 2023-09-24 at 13 12 42](https://user-images.githubusercontent.com/145204716/270158018-6a058a46-df62-424b-91c7-d19e4a84b4ac.jpeg)

**It looks like just all those settings need inputting and your good to go**

**Comment by BarektE** [comment]
We solve it. It can also be done with CLIP, DDT4AL and PyRen.

https://youtu.be/tKro_OGZtRM

https://www.youtube.com/@AtuMOtOElektroniK/search?query=R-Link

**Reply by raymondio6709** [reply]
waste only sharin taht u can do it no information regards how doing it main purpose of this site is sharing not advertising
